index.js

```diff
@@ -11,7 +12,10 @@ class AggregateError extends Error {
 		}
 
 		errors = [...errors].map(error => {
-			if (error instanceof Error) {
+			// Unfortunately stack is not standardized as a property of Error instances
+			// which makes it necessary to explicitly check for it and its type.
+			// Error instances without a stack should be handled as plain objects
+			if (error instanceof Error && isString(error.stack)) {
```

Comment:
`typeof value === 'string'` is enough. The stack can never be a Object-wrapped string.

Reply:
Also, this way, it will lose information about error type. I would consider a better way to handle this.

Reply:
didn’t know that github keeps reviewed but outdated code in place. see below for an answer.